content/en/docs/setup/install/operator/index.md

```diff
@@ -239,76 +239,76 @@ $ kubectl get pods --namespace istio-system \
 
 The process for canary upgrade is similar to the [canary upgrade with `istioctl`](/docs/setup/upgrade/canary/).
 
-For example, to upgrade the revision of Istio installed in the previous section, first verify that the `IstioOperator` CR named `example-istiocontrolplane` exists in your cluster:
+For example, to upgrade Istio {{< istio_previous_version >}}.0 to {{< istio_full_version >}}, first install {{< istio_previous_version >}}.0 and verify that the `IstioOperator` CR named `example-istiocontrolplane` exists in your cluster:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=verify_operator_cr >}}
 $ kubectl get iop --all-namespaces
 NAMESPACE      NAME                        REVISION   STATUS    AGE
 istio-system   example-istiocontrolplane              HEALTHY   11m
 {{< /text >}}
 
 Download and extract the `istioctl` corresponding to the version of Istio you wish to upgrade to.
 Then, run the following command to install the new target revision of the Istio control plane based on the in-cluster
-`IstioOperator` CR (here, we assume the target revision is 1.8.1):
+`IstioOperator` CR (here, we assume the target revision is {{< istio_full_version_revision >}}):
 
-{{< text bash >}}
-$ istio-1.8.1/bin/istioctl operator init --revision 1-8-1
+{{< text syntax=bash snip_id=canary_upgrade_init >}}
+$ istio-{{< istio_full_version >}}/bin/istioctl operator init --revision {{< istio_full_version_revision >}}
 {{< /text >}}
 
 {{< tip >}}
 You can alternatively use Helm to deploy another operator with a different revision setting:
 
-{{< text bash >}}
+{{< text syntax=bash snip_id=canary_upgrade_helm_install >}}
 $ helm install istio-operator manifests/charts/istio-operator \
   --set watchedNamespaces=istio-system \
   -n istio-operator \
-  --set revision=1-9-0
+  --set revision={{< istio_previous_version_revision >}}-1
```

Comment:
I think this should be the same new revision we're upgrading to in the example.
```suggestion
  --set revision={{< istio_full_version_revision >}}
```

Reply:
this was addressed in cfb95bf532521e3983b59f40477d8df9afc415ad